openmetadata-ui/src/main/resources/ui/src/axiosAPIs/ingestionPipelineAPI.ts

```diff
@@ -123,9 +149,14 @@ export const getPipelineServiceHostIp = async () => {
 };
 
 export const getIngestionPipelineLogById = (
-  id: string
+  id: string,
+  after?: string
```

Comment:
isn't it good to add all the params directly instead of `after` so that in future if we need to use any other param except after we don't have to declare them sepratly?

Reply:
@ShaileshParmar11 , For now, we have that structure on the API side, we only have the `after` attribute.